microsoft-edge/toc.yml

```diff
@@ -459,12 +459,22 @@
               href: webview2/get-started/wpf.md
         - name: Sample code
           href: webview2/code-samples-links.md
-        - name: Concepts
+        - name: How to
           items:
+            - name: Automate with Edge Driver
+              href: webview2/how-to/webdriver.md
+            - name: Debug WebView2
+              href: webview2/how-to/debug.md
+            - name: Development best practices
```

Comment:
To avoid unnecessary change, can we just have the two lists appended, rather than resorted? In the short term I think it will just be confusing to have docs moving around, especially if we are planning more moves or grouping in the near future.

Reply:
Finding: Alphabetical TOC is used in these special scenarios:
* Auto-gen'd API Ref.  https://docs.microsoft.com/en-us/microsoft-edge/webview2/webview2-api-reference  User is likely to look up in alpha order.
* List of tools, which matches the order in the Command Menu (separate groups for Panel & Drawer tools) & More Tools menu (interleaved Panel & Drawer tools).  https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/about-panel-drawer-tools

Reply:
Added work item to the backlog to address grouping. To reduce confusion, I'll proceed to sort alphabetically in the meantime.